test/sass/test.scss

```diff
@@ -1,16 +1,8 @@
 /* Requirements */
 @use "../../node_modules/sass-true/sass/true";
-
-@use "../../sass/vars/mdn-web-docs-palette";
-@use "../../sass/vars/layout";
-@use "../../sass/vars/typography" as *;
-
-@use "../../sass/mixins/buttons" as *;
-@use "../../sass/mixins/links";
-@use "../../sass/mixins/typography" as *;
-@use "../../sass/mixins/utils";
-
-@use "../../sass/atoms/buttons" as *;
+@use "../../sass/vars/" as *;
```

Comment:
This is why [index files](https://sass-lang.com/documentation/at-rules/use#index-files) are super nice! We can simply import the entire directory of modules rather than worrying about using the "expected" ones which sometimes can be a pain to keep track of.